website/content/v1.5/talos-guides/configuration/nvidia-gpu-proprietary.md

```diff
@@ -64,7 +64,7 @@ COPY --from=ghcr.io/talos-user/kernel:{{< release >}}-nvidia /boot/vmlinuz /usr/
 Now build the image and push it to the registry.
 
 ```bash
-DOCKER_BUILDKIT=0 docker build --squash --build-arg RM="/lib/modules" -t ghcr.io/talos-user/installer:{{< release >}}-nvidia .
+DOCKER_BUILDKIT=0 docker build --squash -t ghcr.io/talos-user/installer:{{< release >}}-nvidia .
```

Comment:
we should not remove modules shipped with the kernel built, should only use that when using a custom kernel only